### ISSUE TITLE
Add performance summary to UI

### ISSUE THREAD

**Issue by GeorgiChochov** [AbsaOSS/enceladus#48]
It would be helpful to add some performance metrics to Run screen, such as:

* input data size (in GB)
* output data size (in GB)
* Time it took to do standardization
* Time it took to do dynamic conformance
* Total processing time
* For each checkpoint it would be helpful to show the elapsed time as a difference between start and end processing time, if available (for source and raw it may not be available).
* Maybe also the number of records (they already shown as control measurements), could be part of performance summary as well.

An input data size and an output data size should be provided by backend. All other information can be derived from control checkpoints i guess.
There is a generic key/map pair in _INFO file. I guess we can use that to provide information about size of raw, standardized and published datasets. Also, possibly it should include information about standardization and conformance elapsed time.

The raw folder size, std folder size and conformed folder size can be saved in metadata fields of an _INFO file and Run object during std/dynconf jobs execution.

**Comment by GeorgiChochov** [comment]
@DzMakatun How much of this is already done in the `Monitoring` tab?

**Comment by DzMakatun** [comment]
@GeorgiChochov For now, the monitoring focuses on count of succeeded / failed / total  records. It also shows aggregate input/std/output folder size for all runs within info date interval. So the first 2 and the last bullets are addressed in monitoring already. 
- The durations of checkpoints are already shown in the Runs view, as far as I understand. 
- I would like to consider  the run durations VS requested Spark resources (i.e. Performance monitoring) as the next step. I think this should be separate from the Runs view. I would, perhaps,  split the Monitoring into 1. Data Monitoring and 2. Performance Monitoring.

**Comment by GeorgiChochov** [comment]
@DzMakatun Thanks!
Closing as this is already outdated.

**Comment by yruslan** [comment]
This issue is meant to describe what enhancements we can do to the Run metadata page. Currently, it displays a table of key-value pairs. But this exact information can be presented more user-friendly.

**Comment by GeorgiChochov** [comment]
Fair point, let's come back to this and see if we can consolidate some of the stuff from Run Details and Monitoring.

**Comment by yruslan** [comment]
Note for myself - the additional info table is sorted by key in descending order. Ascending order might be better.
